osquery/core/tests/posix/permissions_tests.cpp

```diff
@@ -91,13 +91,13 @@ TEST_F(PermissionsTests, test_path_drop) {
     EXPECT_TRUE(dropper->dropped_);
     EXPECT_EQ(dropper->to_user_, nobody->pw_uid);
 
+    // Dropping "up" to root should fail.
     // Even though this is possible and may make sense, it is confusing!
-    EXPECT_FALSE(dropper->dropTo(getuid(), getgid()));
+    EXPECT_FALSE(dropper->dropTo(0, 0));
```

Comment:
Because only the change works on all systems, the first doesn't work as root on OS X.